conifer/model.py

```diff
@@ -561,4 +561,17 @@ def load_model(filename, new_config=None):
 
     model = make_model(js, config)
     model._metadata = metadata + model._metadata
+
+    try:
+        if config["backend"] in ["cpp","xilinxhls"]:
```

Comment:
Could you find a way to move this code to the backend classes? I want to avoid references to specific backends in this module

Reply:
Yes, I knew that inserting a reference to the backends here is not the clearest solution.

The problem is that the backend classes are not aware if the model is being loaded or created and to make everything work we would need to pass this information to the backend classes (that is also not really clean)
But I have an idea, let me work on that.